routes/transaction.go

```diff
@@ -89,7 +89,7 @@ func (fes *APIServer) GetTxn(ww http.ResponseWriter, req *http.Request) {
 	// committed tip height. Otherwise we'll be missing ~2 blocks in limbo.
 	coreChainTipHeight := fes.TXIndex.CoreChain.BlockTip().Height
 	for fes.TXIndex.TXIndexChain.BlockTip().Height < coreChainTipHeight {
-		if time.Since(startTime) > 30*time.Second {
+		if time.Since(startTime) > 90*time.Second {
```

Comment:
Please revert to 30s. I don't want a bunch of requests hanging around for 90s - it could be bad for the node.